checkQC/parsers/interop_parser.py

```diff
@@ -68,7 +68,7 @@ def run(self):
                 self._send_to_subscribers(("error_rate",
                                         {"lane": lane+1, "read": read_nbr+1, "error_rate": error_rate}))
                 self._send_to_subscribers(("percent_q30",
-                                        {"lane": lane+1, "read": read_nbr+1, "percent_q30": percent_q30, "is_index_read":is_index_read}))
+                                        {"lane": lane+1, "read": read_nbr+1, "percent_q30": percent_q30, "index":read_nbr+1 if is_index_read else "-"}))
```

Comment:
Given my other comment this should be changed back to `"is_index_read": is_index_read`